### ISSUE TITLE
Unable to set `ike_version` on IPSec Tunnels

### ISSUE THREAD

**Issue by kostasns** [terraform-providers/terraform-provider-oci#931]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version and Provider Version

Terraform v0.11.14
+ provider.oci v3.52.0

### Affected Resource(s)

`oci_core_ipsec_connection_tunnel_management`

### Terraform Configuration Files

```hcl
"oci_core_ipsec_connection_tunnel_management" "test_ip_sec_connection_tunnel" {
    #Required
    ipsec_id = "${oci_core_ipsec.test_ipsec.id}"
    tunnel_id = "${oci_core_tunnel.test_tunnel.id}"
    routing = "STATIC"
    display_name = "ipsec_tunnel"
    ike_version = "V2"
}

```

### Expected Behavior

Expect resource to set correct IKE version for specified tunnel

### Actual Behavior

Tunnels remain set with IKE v1.

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

It seems that although [argument is present in the schema](https://github.com/terraform-providers/terraform-provider-oci/blob/471d2f1c75aa13c1f62df8e383574bc500515ad4/oci/core_ipsec_connection_tunnel_management_resource.go#L76) it is not referenced anywhere else in the code.

**Comment by afedorch** [comment]
@kostasns thanks for reporting, we will work on it.

**Comment by afedorch** [comment]
there is a fix in the latest v3.54.0 provider:
- Support and validation for the ike_version, routing attributes in oci_core_ipsec_connection_tunnel_management resource
Thanks!

**Comment by kostasns** [comment]
That was fast! :) Thank you!
